hardware_interface/include/hardware_interface/joint_state_interface.h

```diff
@@ -41,16 +41,92 @@ namespace hardware_interface
 class JointStateHandle
 {
 public:
-  JointStateHandle() : name_(), pos_(0), vel_(0), eff_(0) {}
+  JointStateHandle() : name_(), pos_(0), vel_(0), eff_(0), absolute_pos_(0), torque_sensor_(0) {}
 
   /**
    * \param name The name of the joint
    * \param pos A pointer to the storage for this joint's position
    * \param vel A pointer to the storage for this joint's velocity
    * \param eff A pointer to the storage for this joint's effort (force or torque)
    */
+
+  JointStateHandle(const std::string& name, const double* pos, const double* vel, const double* eff,
+                   const double* absolute_pos, const double *torque_sensor)
+    : name_(name), pos_(pos), vel_(vel), eff_(eff), absolute_pos_(absolute_pos), torque_sensor_(torque_sensor)
+  {
+
```

Comment:
Extra newline at beginning of all these constructors.

Reply:
too much copy pasta